MapboxDirections/MBCongestion.h

```diff
@@ -0,0 +1,12 @@
+typedef NS_OPTIONS(NSUInteger, MBCongestion) {
```

Comment:
This should be called MBCongestionLevel, for consistency with MBAttributeCongestionLevel.